examples/language-features/sets.qnt

```diff
@@ -0,0 +1,35 @@
+module sets {
```

Comment:
```suggestion
// -*- mode: Bluespec; -*-
module sets {
```

A simple hack to make specs a bit more readable

Reply:
IWhere does this work?